performance_test/script.mjs

```diff
@@ -0,0 +1,98 @@
+import { execSync } from "child_process";
+import { copyFile } from "fs/promises";
+import { performance } from "perf_hooks";
+import { existsSync, mkdirSync, rmdirSync } from "fs";
+
+const DEFAULT_NUMBER_OF_FILE = 5;
+const DEFAULT_ITERATION = 1;
+const BASE_PATH = "./performance_test";
```

Comment:
use __filename

Reply:
done. since the script switched to typescript. i used __dirname instead.